model/src/main/proto/profile.proto

```diff
@@ -108,7 +108,7 @@ enum StoryTextSize {
 // TODO(#669): Add translation for language selection and track updating of this list to match the supported translation list.
 // Supported App language for user preference.
 enum AppLanguage {
-  ENGLISH_APP_LANGUAGE = 0;
+  ENGLISH_APP_LANGUAGE_UNSPECIFIED = 0;
```

Comment:
So, this `buf` linter add _UNSPECIFIED as a suffix to all those whose values are 0.

Reply:
Actually what we should do is this:
```
enum AppLanguage {
APP_LANGUAGE_UNSPECIFIED = 0;
ENGLISH_APP_LANGUAGE = 1;
}
```

In general for protos, avoid giving actual values in the 0th position. Let 0 always have a 'ENUM_NAME_UNSPECIFIED' as the value. This is because, if we do not set the value for an enum, proto by default sets the value as 0. And we would want to differentiate between the case where the value is a default value, or actually set..vs a value has not been set (so it is 0)

Hope this makes sense!

Reply:
ok

Reply:
What about `topic.proto` completion state?

```
enum ChapterPlayState {
  // The completion status is unknown.
  COMPLETION_STATUS_UNSPECIFIED = 0;
}
```

this can be change to `CHAPTER_PLAY_STATE_UNSPECIFIED =0;`

Reply:
I will defer to @rt4914 for this. @rt4914 what is the right thing to do for chapter play state? Or who should we ask?

Reply:
@anandwana001 

```
// Represents the play state of a single chapter.
enum ChapterPlayState {
  // The completion status is unknown.
 CHAPTER_PLAY_STATE_UNSPECIFIED = 0;

  // The chapter has not yet been started, but can be started by the player.
  NOT_STARTED = 1;

  // The chapter has not yet been started, and cannot be started since the player is missing prerequisites.
  NOT_PLAYABLE_MISSING_PREREQUISITES = 2;

  // The chapter has been started but not completed by the player.
  STARTED_NOT_COMPLETED = 3;

  // The chapter has been completed by the player.
  COMPLETED = 4;
}
```

```
// Supported Story text sizes for user preference.
enum StoryTextSize {
  TEXT_SIZE_UNSPECIFIED = 0;
  SMALL_TEXT_SIZE = 1;
  MEDIUM_TEXT_SIZE = 2;
  LARGE_TEXT_SIZE = 3;
  EXTRA_LARGE_TEXT_SIZE =  4;
}
```

```
// TODO(#669): Add translation for language selection and track updating of this list to match the supported translation list.
// Supported App language for user preference.
enum AppLanguage {
  APP_LANGUAGE_UNSPECIFIED = 0;
  ENGLISH_APP_LANGUAGE = 1;
  HINDI_APP_LANGUAGE = 2;
  FRENCH_APP_LANGUAGE = 3;
  CHINESE_APP_LANGUAGE =  4;
}
```

```
// TODO(#669): Add translation for language selection and track updating of this list to match the supported translation list.
// Supported Audio language for user preference.
enum AudioLanguage {
  AUDIO_LANGUAGE_UNSPECIFIED = 0;
  NO_AUDIO = 1;
  ENGLISH_AUDIO_LANGUAGE = 2;
  HINDI_AUDIO_LANGUAGE = 3;
  FRENCH_AUDIO_LANGUAGE = 4;
  CHINESE_AUDIO_LANGUAGE =  5;
}
```


@anandwana001 Check this out. I think this should work for you as well for the previous implementation. I haven't run the code, so it gives some error let me know.

Reply:
@vinitamurthi @anandwana001 I think the above comment should work and even if it does not work, just let me know I will check and update the code accordingly because the above code has some consistency to it which would be nice to follow in entire project.

Reply:
`exploration.proto`

```
enum ObjectType {
  OBJECT_TYPE_UNSPECIFIED = 0;
  UNICODE_STRING = 1;
}
```

@BenHenning Could we make this update in `exploration.proto`, or will it affect any of your other implementations?